src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -411,11 +411,7 @@ public String getInvalidityInfoForSectionName(String sectionName) {
      * @return An explanation of why the {@code teamName} is not acceptable.
      *         Returns an empty string if the {@code teamName} is acceptable.
      */
-    public String getInvalidityInfoForTeamName(String teamName) {
-        boolean isValidEmail = StringHelper.isMatching(teamName, REGEX_EMAIL);
-        if (isValidEmail) {
-            return TEAM_NAME_IS_VALID_EMAIL_ERROR_MESSAGE;
-        }
+
         return getValidityInfoForAllowedName(TEAM_NAME_FIELD_NAME, TEAM_NAME_MAX_LENGTH, teamName);
```

Comment:
Not quite sure what the change is here. If not applicable, revert it

Reply:
Yep, this is also the reason for the failing Travis builds

Reply:
@RonakLakhotia yeah I had accidentally deleted that whole statement instead of editing it :(